mne/io/pick.py

```diff
@@ -60,7 +60,11 @@ def channel_type(info, idx):
     elif kind == FIFF.FIFFV_SYST_CH:
         return 'syst'
     elif kind in [FIFF.FIFFV_SEEG_CH, 702]:  # 702 was used before v0.11
```

Comment:
Or is it so that it should be removed for 13? I don't the warning message got to release 11. I think the warning in https://github.com/mne-tools/mne-python/blob/master/mne/viz/raw.py#L249 should be changed to 13.